app/src/main/java/com/endava/myendava/fragments/CalendarFragment.java

```diff
@@ -0,0 +1,152 @@
+        View view = inflater.inflate(R.layout.fragment_calendar, parent, false);
+        setupModule();
+        return view;
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        mUnbinder = ButterKnife.bind(this, view);
+        setMonthButton();
+        setupCalendarDayAdapter();
+        setupParentEventsAdapter();
+    }
+
+    private void setMonthButton() {
+        Calendar calendar = Calendar.getInstance();
+        SimpleDateFormat month_date = new SimpleDateFormat("MMMM");
+        String month_name = month_date.format(calendar.getTime());
+        mCalendarMonthButton.setText(month_name + " " + calendar.get(Calendar.YEAR));
+    }
+
+    private void setupModule() {
+        ApplicationServiceLocator locator = (ApplicationServiceLocator) Objects.requireNonNull(getActivity()).getApplicationContext();
+        locator.getDashboardComponent(this).inject(this);
+    }
+
+    private void setupParentEventsAdapter() {
+        mParentEventAdapter = new EventParentAdapter(getContext(), getEventParentList());
+        mProjectsRecyclerView.setAdapter(mParentEventAdapter);
+        mProjectsRecyclerView.setLayoutManager(new LinearLayoutManager(getActivity()));
+    }
+
+    private void setupCalendarDayAdapter() {
+        mCalendarDayAdapter = new CalendarDayAdapter(getContext(), getCalendarDayList());
+        mCalendarRecycleView.setAdapter(mCalendarDayAdapter);
+        LinearLayoutManager layoutManager = new LinearLayoutManager(getContext(), LinearLayoutManager.HORIZONTAL, false);
+        mCalendarRecycleView.setLayoutManager(layoutManager);
+    }
+
+    private List<EventParent> getEventParentList() {
+        EventParent eventParent1 = new EventParent("30 OCTOBER", getEventList().get(0));
+        EventParent eventParent2 = new EventParent("31 OCTOBER", getEventList().get(1));
+        EventParent eventParent3 = new EventParent("1 NOVEMBER", getEventList().get(2));
+        return new ArrayList<>(Arrays.asList(eventParent1, eventParent2, eventParent3));
+    }
+
+    private List<List<Event>> getEventList() {
+        List<Tag> tags = new ArrayList<>(Arrays.asList(new Tag("", "", "", 2, "Android", ""),
+                new Tag("", "", "", 2, "MVP", "")));
+        Event event1 = new Event("Training title", "2h 30", tags, 4, "Training");
+        Event event2 = new Event("Certification title", "1h 30", tags, 2, "Certification");
+        Event event3 = new Event("Conference title", "2h", tags, 9, "Conference");
+        Event event4 = new Event("Training title", "3h 30", tags, 5, "Training");
+        List<Event> list1 = new ArrayList<>();
+        List<Event> list2 = new ArrayList<>(Arrays.asList(event1, event2, event3));
+        List<Event> list3 = new ArrayList<>(Arrays.asList(event1, event2, event3, event4));
+        return new ArrayList<>(Arrays.asList(list1, list2, list3));
+    }
+
+    private List<CalendarDay> getCalendarDayList() {
```

Comment:
move data generation to another file which can be easily deleted after real implementation